Sources/Emoji.swift

```diff
@@ -10,7 +10,7 @@ public final class Emoji : EmojiData {
     ///
     public static var SingleEmojiPattern:String = {
         // The "emoji" group needs to be followed by a special character to be rendered like emoji.
-        let emojiVariants = "(?:(?:\(EmojiPatterns.joinWithSeparator("|")))\\uFE0F)"
+        let emojiVariants = "(?:(?:\(EmojiPatterns.joinWithSeparator("|")))\\uFE0F{0,1})"
```

Comment:
What was the use case that inspired this change? My worry is that `EmojiPatterns` contains characters like the digits 0-9, which this pattern would then match. The `FEOF` marks those characters as Emoji, but without it should be skipped.